playwright-tests/tests/treasury-factory/my-treasuries.spec.js

```diff
@@ -0,0 +1,49 @@
+import { expect } from "@playwright/test";
+import { test } from "../../util/test.js";
+import { mockUserDaos } from "../../util/rpcmock.js";
+
+test.afterEach(async ({ page }, testInfo) => {
+  console.log(`Finished ${testInfo.title} with status ${testInfo.status}`);
+  await page.unrouteAll({ behavior: "ignoreErrors" });
+});
+
+test.describe("My Treasuries", () => {
+  test.use({
+    storageState: "playwright-tests/storage-states/wallet-connected-admin.json",
+  });
+
+  test("Should navigate to create treasury page when user don't have any existing DAOs", async ({
+    page,
+    factoryAccount,
+  }) => {
+    test.setTimeout(120_000);
+    await mockUserDaos({ page, accountId: "megha19.near" });
+    await page.goto(`/${factoryAccount}/widget/app`);
+    await page.waitForTimeout(3_000);
+    await expect(
+      page.getByRole("heading", { name: "Treasury Creation" }),
+    ).toBeVisible();
+  });
+
+  test("Should show user treasuries and DAOs", async ({
```

Comment:
so this applies when user has existing DAOs?

Reply:
yes, updated the description